sql/core/src/main/scala/org/apache/spark/sql/execution/python/PythonArrowOutput.scala

```diff
@@ -37,6 +37,8 @@ import org.apache.spark.sql.vectorized.{ArrowColumnVector, ColumnarBatch, Column
  */
 private[python] trait PythonArrowOutput { self: BasePythonRunner[_, ColumnarBatch] =>
 
+  protected def handleMetadataAfterExec(stream: DataInputStream): Unit = { }
+
```

Comment:
By default we don't use it. I don't see it is used in other place too. Do you have any plan for it?

Reply:
Oh yeah I do, see https://github.com/apache/spark/compare/master...HeartSaVioR:spark:WIP-flatmapgroupswithstate-pyspark